doc/GAME_BALANCE.md

```diff
@@ -97,6 +97,48 @@ A measure of how well-suited the item is for being swung/thrust/etc. This factor
 
 +1 - Well-balanced for swinging or stabbing.  Baseball bat, golf club, swords, quarterstaff, knives.
 
+##Damage
+Weapon's relative strength is based on an approximate formula involving its damage, to-hit, techniques and few other factors.
+This strength will be expressed in "points" or dpt (damage per turn) as damage per turn is the most significant balancing factor.
+
+### Damage per turn
+Melee weapon's relative strength is measured by damage per turn (more precisely: damage per 100 moves). The damage is sum of all damage types (average of min/max where random) from the weapon alone, without including strength in the calculation.
+For bashing weapons, it is assumed the character can achieve the maximum damage, but this strength isn't added to damage (or critical damage).
+
+### To-hit
+While not a direct measure of damage, to-hit bonus has to be included in the calculations, as it has a significant effect on actual damage output (through hit and crit rate).
+
+Each point of to-hit is "worth" roughly a 10% increase/decrease in damage per turn or 2.5 points of damage per turn (whichever is higher).
+
+### Techniques
+Rapid strike technique increases damage per turn by 0.66 / 0.5 = 132%
+Other techniques are generally too situational to be worth any points here.
+
+### Weapon tiers
+Damage per turn should put the weapon into one of those categories:
+
+<10 - Not weapons. Those items may be pressed into service, but are unlikely to be better than fists. Plastic bottles, rocks, boots.
+
+11-15 - Tools not meant to strike and improvised weapons. Two-by-fours, pointy sticks, pipes, hammers.
+
+16-25 - Dangerous tools or weak dedicated weapons. Golf clubs, two-by-swords, wooden spears, knife spears, hatchets, switchblades, tonfas, quarterstaves.
+
+26-35 - The most dangerous of tools that aren't made to be weapons. Wood and fire axes, steel spears, electric carvers, kukris, bokken, machetes, barbed wire bats.
```

Comment:
Nitpicking here, but I'd make this description include "improvised weapons", as a barbed wire bat is clearly made to be a weapon.  Could get even more specific and make the distinction between "crudely weapons, improvised weapons, professionally made weapons".